addon/components/base-chart-component.js

```diff
@@ -81,6 +81,17 @@ export default Component.extend({
             this.onClick(d);
         });
 
+        // when user navigates using tab key and uses enter button to drill-in the chart
+        const onClick = this.onClick;
+        this.get('chart').selectAll(this.elementToApplyTipSelector).each(function () {
```

Comment:
Could remove `const onClick` above in favor of preserving `this` with an es6 function here
```suggestion
        this.get('chart').selectAll(this.elementToApplyTipSelector).each(() => {
```

Reply:
And doing `this.onClick(...)` below

Reply:
in the same @each, i'm using `const bar = d3.select(this);` 'this' here is the context of the d3 selection and not of the entire component.

Reply:
![Screen Shot 2019-11-26 at 1 51 12 PM](https://user-images.githubusercontent.com/42894144/69663237-d92a9d00-1053-11ea-9387-d0b6b402a0ed.png)